tools/elk/bin/target

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
```

Comment:
We should probably rename this file as it keeps getting accidentally removed

Reply:
Save the Target!!!